ntb_hw_switchtec.c

```diff
@@ -756,6 +756,22 @@ static int switchtec_ntb_setup_crosslink(struct switchtec_ntb *sndev)
 	return 0;
 }
 
+static void switchtec_ntb_cleanup_crosslink(struct switchtec_ntb *sndev)
+{
+	if (!crosslink_is_enabled(sndev))
+		return;
+
+	if (sndev->mmio_xlink_dbmsg_win) {
+		pci_iounmap(sndev->stdev->pdev, sndev->mmio_xlink_dbmsg_win);
+		sndev->mmio_xlink_dbmsg_win = NULL;
+	}
+
+	if (sndev->mmio_xlink_ctrl_win) {
+		pci_iounmap(sndev->stdev->pdev, sndev->mmio_xlink_ctrl_win);
+		sndev->mmio_xlink_ctrl_win = NULL;
```

Comment:
Could use a better commit message.

Seems like we're doing a cleanup, but actually adding a second call site and the NULL setting because of the new call site.

Reply:
Do you mean the second call site should be moved to separate commit? As these two reserved LUT can be re-initialized, so the NULL setting is added.

Reply:
I'm not sure a second commit is actually necessary here. The patch is pretty small. Just describe the changes better in the commit message.

Reply:
Updated the commit message.